fibonacci-heap.js

```diff
@@ -317,4 +317,6 @@ MaxFibonacciHeap.from = function(iterable, comparator) {
  */
 FibonacciHeap.MinFibonacciHeap = FibonacciHeap;
 FibonacciHeap.MaxFibonacciHeap = MaxFibonacciHeap;
+exports.MinFibonacciHeap = FibonacciHeap;
```

Comment:
I have some doubts here. I remember CommonJS consumers and packagers strictly enforcing `module.exports` or named `exports` but crashing/yelling if both are used?

Reply:
It's a usually a bad practice to mix both. But not because of node.js lack of support or the like. I will remove to mix them in the same file.